integration-tests/tests/run-workflow.js

```diff
@@ -50,23 +58,28 @@ const testRunWorkflowFn = _.flow(
   ])
   // stopLoggingPageAjaxResponses()
 
+  const disablePageLogging = logPageConsoleMessages(page)
+  const start = Date.now()
   await pRetry(async () => {
     try {
       await findInGrid(page, 'Succeeded', { timeout: 65 * 1000 }) // long enough for the submission details to refresh
     } catch (e) {
+      try {
+        await findInGrid(page, 'Running', { timeout: 1000 })
+        console.log(`Workspace is running, elapsed time (minutes): ${(Date.now() - start) / (1000 * 60)}`)
+      } catch (e) {
+        console.log(`Workspace not yet running, elapsed time (minutes): ${(Date.now() - start) / (1000 * 60)}`)
+      }
       throw new Error(e)
     }
-  }, { retries: 10, factor: 1 })
-
-  await click(page, navChild('data'))
-  await click(page, clickable({ textContains: 'test_entity' }))
-  await findInDataTableRow(page, testEntity.name, testEntity.attributes.input)
-})
+  }, { retries: 15, factor: 1 })
```

Comment:
Increases timeout for workflow to finish to 15 minutes (from 10). Note that we hope to ultimately decrease this after we have improved the submission speed to Cromwell. 

To understand better what percentage of the time waiting is in "Submitted" vs. "Running", I added the log messages. In a few of the failures that we have seen, the workflow doesn't even make it to "Running" within the previous 10 minute timeout.

Reply:
Example logging message in test run:

````
page.http.res 200 GET https://rawls.dsde-dev.broadinstitute.org/api/workspaces/saturn-integration-test-dev/test-workspace-22765/snapshots/v2?offset=0&limit=1000
 PASS  jest/1-run-workflow-on-snapshot.integration-test.js (360.182 s)
  ● Console

    console.info
      Workflow not yet running, elapsed time (minutes): 1.1000333333333334

      at pRetry.retries (tests/run-workflow.js:71:17)

    console.info
      Workflow is running, elapsed time (minutes): 2.20035

      at pRetry.retries (tests/run-workflow.js:69:17)
          at runMicrotasks (<anonymous>)

    console.info
      Workflow is running, elapsed time (minutes): 3.3006333333333333

      at pRetry.retries (tests/run-workflow.js:69:17)
          at runMicrotasks (<anonymous>)

Reply:
Is there a way to not print out references to code lines like `at pRetry.retries (tests/run-workflow.js:71:17)`? They don't seem helpful and make the output more verbose.

Reply:
@kyuksel honestly I don't know, but since this is only test output, I don't think it's worth the investigation time.

Reply:
Following up on this, `logPageConsoleMessages` doesn't seem to be intercepting the console.info calls. I will follow up with @dmohs about how I am supposed to use the method.